README.md

```diff
@@ -42,6 +42,8 @@ To add Healthkit support to your application's `Capabilities`
 - Select the project name in the left sidebar
 - In the main view select '+ Capability' and double click 'HealthKit'
 
+> 🚨 Expo: This package is not available in the [Expo Go](https://expo.io/client) app. Learn how you can use it with [custom dev clients](/docs/Expo.md).
```

Comment:
Think it might be valuable to add the Installation instructions for Expo as an alternative from the regular _yarn_/_npm_ one.

Adding that to the first step of this list (`Install the react-native-health package from npm`) would make it more clear and attract more developers to use it.

What do you think?